stable/elasticsearch-exporter/values.yaml

```diff
@@ -51,7 +51,19 @@ service:
 ## env:
 ##   KEY_1: value1
 ##   KEY_2: value2
-# env:
+env: {}
+
+## Create a secret manually. Reference it here to inject environment variables
+## envFromSecrets:
```

Comment:
Do you have a particular reason to go this way instead of what is done in other charts? See https://github.com/helm/charts/search?utf8=%E2%9C%93&q=envFromSecret&type=

Reply:
Actually no, I just followed the way from curator chart.